### DISCUSSION TITLE
Mapsui Editing

### DISCUSSION THREAD

**Discussion by wangkai942901902** [Mapsui/Mapsui#2311] [Q&A]
Mapsui Provides an example, the file name is EditingSample, I want to change the size of the draw point should be changed how?And I want to set points, lines and surfaces in a less style

**Comment by pauldendulk** [comment]
'less' as in [less.js](https://lesscss.org)?  That is not an option in the graphics renderer we use. For point symbols you could use SVGs.

**Reply by wangkai942901902** [reply]
Hello, I used mapsui in the wpf project. I want to set multiple different styles for different paintings in one layer. What should I do? What I've learned so far is that there can only be one style in a layer, and objects in the layer will all use the styles in the layer.

**Reply by pauldendulk** [reply]
You can use a ThemeStyle to apply a different style for different types of features.

**Reply by wangkai942901902** [reply]
thank you!
